tests/integration/test_slurm.py

```diff
@@ -362,3 +362,26 @@ def patched_get_jobs_list(self, jobs=None, user=None, timeout=None):
     assert (
         job_controller.count_flows(states=FlowState.COMPLETED) == 1
     ), f"Flows not marked as completed, full flow info:\n{job_controller.get_flows({})}"
+
+
+@pytest.mark.parametrize(
+    "worker",
+    ["test_sanitize_local_worker", "test_sanitize_remote_worker"],
+)
+def test_sanitize(worker, job_controller):
+    from jobflow import Flow
+
+    from jobflow_remote import submit_flow
+    from jobflow_remote.jobs.runner import Runner
+    from jobflow_remote.jobs.state import JobState
+    from jobflow_remote.testing import add
+
+    assert job_controller.project.workers[worker].get_host().sanitize is True
+
+    flow = Flow([add(1, 2)])
+    submit_flow(flow, worker=worker)
+
+    runner = Runner()
+    runner.run_one_job()
+
+    assert job_controller.count_jobs(states=JobState.COMPLETED) == 1
```

Comment:
The worker here does not have spurious echos here. Would we need something like that ? Then that's another worker that's spawned (or maybe the worker could be modified temporarily ? like modifying the .bashrc or bash profile i don't know and then putting it back to "normal" after the test). Not sure if it is worth anyway, just wondering.